src/main/java/catan/domain/model/game/GameUserBean.java

```diff
@@ -43,6 +42,9 @@
     @Column(name = "MOVE_ORDER", unique = false, nullable = false)
     private int moveOrder;
 
+    @ManyToOne(cascade = CascadeType.ALL)
```

Comment:
Are you sure that we have to provide ManyToOne relationship here? I think each game user hase only one set of resources and each set of resources can belong only to one gameUser.